testing/xmrig/APKBUILD

```diff
@@ -2,22 +2,28 @@
 # Maintainer: Oleg Titov <oleg.titov@gmail.com>
 
 pkgname=xmrig
-pkgver=2.13.1
+pkgver=2.14.1
 pkgrel=0
 pkgdesc="XMRig is a high performance Monero (XMR) miner"
 url="https://github.com/xmrig/xmrig"
-arch="x86_64 x86"
+arch="x86_64 x86 aarch64 armv7 armhf"
 license="GPL-3.0"
 makedepends="cmake libmicrohttpd-dev libuv-dev openssl-dev"
 subpackages="$pkgname-doc"
 source="$pkgname-$pkgver.tar.gz::https://github.com/xmrig/xmrig/archive/v$pkgver.tar.gz"
 builddir="$srcdir/$pkgname-$pkgver"
 
+case "$CARCH" in
+	aarch64*) ARM_TARGET=8; makedepends="$makedepends linux-headers" ;;
+	armv7*)   ARM_TARGET=7; makedepends="$makedepends linux-headers" ;;
+	armhf*)   ARM_TARGET=7; makedepends="$makedepends linux-headers" ;;
+esac
+
 build() {
 	cd "$builddir"
 	mkdir build
 	cd build
-	cmake ..
+	cmake .. -DARM_TARGET=${ARM_TARGET}
```

Comment:
I'd better made it optional argument like `CMAKE_CROSSOPTS="-DARM_TARGET=8"` and used as `cmake .. ${CMAKE_CROSSOPTS}`

Reply:
Thank you for the tip! I will implement this tomorrow.

Reply:
Done!